platform/saivpp/vpplib/SwitchStateBaseRoute.cpp

```diff
@@ -137,6 +179,82 @@ void create_vpp_nexthop_entry (
     vpp_nexthop->weight = 1;
 }
 
+// Function to convert an IPv4 address from unsigned integer to string representation
+std::string SwitchStateBase::convertIPToString (
```

Comment:
If you use vpp_get_intf_name_for_prefix() then there is no need of these convert functions.
getIpPrefixFromSaiPrefix() used in vpp_get_intf_name_for_prefix() can do better job.

Reply:
Done